src/assets/videos/videos.ts

```diff
@@ -1,12 +1,11 @@
 import { IVideoCard } from "../../common/interfaces/IVideoCard";
 
-
 const videos: IVideoCard[] = [
   {
     url: "",
     title: "",
     clientName: "",
-    genre: "animation",
+    genres: ["animation"],
     roles: ["camera_operator"],
```

Comment:
Is entering things here as snake case going to be the easiest thing for Kurt? It'll help for type checking and validation, but just trying to think forward for our non-programmer friend who will have to remember all of this in 10 years.

Reply:
Good Question.  It.may not be the easiest to remember for him. We could look into changing that.  Good news is that it is type checked, so he'll know immediately if it's wrong. Same is true if it's not snake_case, though. So I'm indifferent.